cerebrium/fine-tuning/auto-deploying.mdx

```diff
@@ -33,11 +27,11 @@ autodeploy:
   hardware: AMPERE_A5000 # Hardware to use for the deployment
   cpu: 2 # Number of CPUs to allocate to the deployment
   memory: 14.5 # Memory to allocate to the deployment. This depends on your model but for most 7B models, 14.5GB is sufficient.
-  autodeploy-requirements-file: /Path/To/Your/requirements.txt # Requirements that you would need in a cortex deployment
-  autodeploy-pkglist-file: /Path/To/Your/pkglist.txt # Optional pkglist that you would need in a cortex deployment
+  autodeploy-requirements-file: /Path/To/Your/requirements.txt # Optional requirements.txt
```

Comment:
Why would they need these? They don't even know the code

Reply:
If a user wants to set a specific version of a library?
It's entirely optional.

Reply:
Erring on the side of giving them more control rather than less.